Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1307,7 +1307,7 @@ private extension PurchasesOrchestrator {
                              completion: @escaping @Sendable (Result<PromotionalOffer, PurchasesError>) -> Void) {
 
         _ = Task<Void, Never> {
-            let transaction = await self.transactionFetcher.firstVerifiedAutoRenewableTransaction
+            let transaction = await self.transactionFetcher.firstVerifiedTransaction
```

Comment:
Should this only be subscriptions? Not sure if we can have promotional purchases for non-subscriptions?

Reply:
Changed back to return only subscriptions for promotional offers 👍